app/src/main/res/values-fr/strings.xml

```diff
@@ -56,7 +56,7 @@
     <string name="settings_backup_apk_dialog_cancel">Annuler</string>
     <string name="settings_backup_apk_title">Sauvegarde de l’appli</string>
     <string name="settings_category_app_data_backup">Sauvegarde des données de l’appli</string>
-    <string name="settings_auto_restore_summary_usb">Remarque : votre %1$ doit être branché pour que ceci fonctionne.</string>
+    <string name="settings_auto_restore_summary_usb">Remarque : votre %1$s doit être branché pour que ceci fonctionne.</string>
```

Comment:
@grote look!

I fixed this one manually to verify the check - it wouldn't let me save the wrong version.
Not sure why it's showing a difference in the space at the front though, I was using that to test the checks but it should've been the same again.

Reply:
Great!

Maybe the French have different white-spaces? :P

Reply:
Looks like they used mixed normal space and NO-BREAK SPACE in this file